platform-sdk/swirlds-state-api/src/test/java/com/swirlds/state/spi/FilteredWritableStatesTest.java

```diff
@@ -91,37 +85,27 @@ void empty() {
         @Test
         @DisplayName("Contains")
         void contains() {
-            assertThat(states.contains(FRUIT_STATE_KEY)).isFalse();
-        }
-
-        @Test
-        @DisplayName("Throws NPE if the key is null")
-        void nullKey() {
-            //noinspection DataFlowIssue
-            assertThatThrownBy(() -> states.get(null)).isInstanceOf(NullPointerException.class);
+            assertThat(states.contains(FRUIT_STATE_ID)).isFalse();
         }
 
         @Test
         @DisplayName("Throws IAE for any non-null key")
```

Comment:
Seems like method and display names for 2 tests below should be updated:
```suggestion
    @DisplayName("Throws IAE for any unknown ID")
```